src/simsoptpp/curveplanarfourier.cpp

```diff
@@ -16,10 +16,9 @@ void CurvePlanarFourier<Array>::gamma_impl(Array& data, Array& quadpoints) {
     double sinphi, cosphi, siniphi, cosiphi;
     int numquadpoints = quadpoints.size();
     data *= 0;
-
     Array q_norm = q * inv_magnitude();
 
-
+// #pragma omp parallel for schedule(static) 
```

Comment:
This file has many commented-out omp directives - generally I'd suggest removing commented-out code like this.